DistFiles/xMatter/bloom-xmatter-mixins.jade

```diff
@@ -64,11 +64,9 @@ mixin block-licenseAndCopyright
 		.copyright(data-book="copyright", lang="*").Credits-Page-style
 			{copyright}
 		.licenseBlock
-			img.licenseImage(src="" data-book="licenseImage")
-			.licenseUrl(data-book="licenseUrl", lang="en")
-				{License Url}
-			.licenseDescription(data-book="licenseDescription",lang="en").Credits-Page-style
-				{License Description}
+			img.licenseImage(src="license.png" data-book="licenseImage")
+			.licenseUrl(data-book="licenseUrl", lang="en") http://creativecommons.org/licenses/by/4.0/
+			.licenseDescription(data-book="licenseDescription",lang="en").Credits-Page-style You are free to make commercial use of this work. You may adapt and add to this work. You must keep the copyright and credits for authors, illustrators, etc.
```

Comment:
Should we do something about localizing this? I don't know whether we do when the user selects this license in the UI (and the relevant book language is not English), but if so, we should probably make sure that the right localized version comes up by default also.